var/spack/repos/builtin/packages/brayns/package.py

```diff
@@ -44,20 +45,20 @@ class Brayns(CMakePackage):
     depends_on('assimp@4.1.0', when='@:1.1.0 +assimp')
     depends_on('bbptestdata', type='test', when='+test')
     depends_on('brion', when='+brion')
-    depends_on('deflect ~deflect-qt', when='+deflect')
-    depends_on('freeimage')
+    depends_on('deflect ~deflect-qt', when='@:1.1.0 +deflect')
```

Comment:
Awesome!

I'm wondering if we could move all 1.x dependencies to a block below current dependencies? That would make purging them once 1.0 is deprecated so much easier :)

Reply:
Good point, done!

Reply:
Thanks!